doc-content/enterprise-only/getting-started-kogito/ref-kogito-microservice-deploy-troubleshooting.adoc

```diff
@@ -0,0 +1,22 @@
+[id="ref-kogito-microservice-deploy-troubleshooting_{context}"]
+= {KOGITO} microservice deployment troubleshooting
+
+Use the information in this section to troubleshoot issues that you might encounter when using the {KOGITO} Operator to deploy {KOGITO} microservices. The following information is updated as new issues and workarounds are discovered.
```

Comment:
```suggestion
Use the information in this section to troubleshoot issues you might encounter when using the {KOGITO} Operator to deploy {KOGITO} microservices. The following information is updated as new issues, and workarounds are discovered.
```

"is updated" - passive voice.
"are discovered" - passive voice.

Reply:
Just remove the comma.i think everything else is OK.